packages/api-explorer/src/components/DocMethodSummary/utils.tsx

```diff
@@ -27,7 +27,7 @@ import React from 'react'
 import styled from 'styled-components'
 import { Icon, Status } from '@looker/components'
 import { Beaker } from '@looker/icons'
-import { VisibilityOff } from '@styled-icons/material'
+import { VisibilityOff } from '@styled-icons/material/VisibilityOff'
```

Comment:
Interesting the deeper import is required. Did you fix this with `--fix`?

Reply:
Unfortunately not :( . That one was a bit painful

Reply:
Ouch indeed